WalletWasabi.Fluent/Controls/PrivacyContentControl.axaml

```diff
@@ -0,0 +1,61 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:controls="using:WalletWasabi.Fluent.Controls"
+        xmlns:conv="clr-namespace:WalletWasabi.Fluent.Converters">
+  <Design.PreviewWith>
+    <controls:PrivacyContentControl />
+  </Design.PreviewWith>
+
+  <Style Selector="controls|PrivacyContentControl">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Panel Background="Transparent">
+          <ContentPresenter Name="PART_Content"
+                            Opacity="{TemplateBinding IsContentVisible, Converter={x:Static conv:BoolOpacityConverters.BoolToOpacity}}"
+                            IsVisible="{Binding $self.Opacity, Converter={x:Static conv:BoolOpacityConverters.OpacityToBool}}"
```

Comment:
Can you just use directly binding to IsContentVisible here?

Reply:
It is because of the animation

Reply:
fixed, did the animation in another way